Plugin.php

```diff
@@ -70,6 +76,11 @@ public function boot()
                     ) {
                         $validation = Validator::make(post(), ['password' => 'notpwned']);
                         if ($validation->fails()) {
+                            Event::fire('pwnedpasswords.backend.login_rejected', [[
+                                'time' => now()->toDateTimeString(),
+                                'action' => $action,
+                                'user_input' => post('login') ?? post('email') ?? '(unknown)',
+                            ]]);
```

Comment:
This should go inside of the login event listener so that it only fires when the login was successful with an invalid password. We don't need to provide the time as the listener can do that, and the user_input can then be grabbed directly from the user object itself. We can probably just pass the user object itself as context in the event. So `Event::fire('winter.pwnedpasswords.backend.login_rejected', [$user]);`.